Automattic/jetpack
# Like block: Add post likes API requests to front-end and editor; render avatar of one liker

> [!NOTE]  
> This PR is not meant to be reviewed or merged. It is an exploration into looking at how the Like block could be build without the current `iframe` implementation.

## Proposed changes:

* introduce `fetch_likes_data` to get likers on the server side (for the front-end block render)
* introduce `useFetchPostLikes` to get likers in the block editor
* render one liker in the editor and front-end

### Other information:


## Jetpack product discussion



Related project thread: pdDOJh-2JP-p2

## Does this pull request change what data or activity we track or use?



No, it does not.

## Testing instructions:

1. Check out the PR and build it.
2. Since this new block is in beta, make sure your WordPress site has `define( 'JETPACK_BLOCKS_VARIATION', 'beta' );` line in its `wp-config.php`.
3. Launch Jurassic Tube to make your local site available publicly.
4. Connect Jetpack to your WordPress.com account and enable Likes feature.
5. Add the new Like block to a post on your test site.
6. Use the existing (old) Like button to like the post. When you refresh the post, your Gravatar should appear at the place of the new block (same applies to the block editor).
